microservices/matchmaking-service/src/scrim/scrim.consumer.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    InjectQueue, OnQueueFailed, Process, Processor,
+} from "@nestjs/bull";
+import {Logger} from "@nestjs/common";
+import {ScrimStatus} from "@sprocketbot/common";
+import {Job, Queue} from "bull";
+import {compareAsc} from "date-fns";
+import {now} from "lodash";
+
+import {ScrimService} from "./scrim.service";
+import {ScrimCrudService} from "./scrim-crud/scrim-crud.service";
+
+export const MATCHMAKING_QUEUE = "matchmaking";
+const SCRIM_CLOCK_JOB = "scrimClockJob";
+
+@Processor(MATCHMAKING_QUEUE)
+export class ScrimConsumer {
+    private readonly logger = new Logger(ScrimConsumer.name);
+
+    constructor(
+        @InjectQueue(MATCHMAKING_QUEUE) private eloQueue: Queue,
```

Comment:
We should rename this to be matchmakingQueue or just queue instead of eloQueue

Reply:
Missed that change.